examples/log.c

```diff
@@ -424,7 +429,7 @@ static int parse_options(
 			else
 				/** Try failed revision parse as filename. */
 				break;
-		} else if (!match_arg_separator(&args)) {
+		} else if (match_arg_separator(&args)) {
```

Comment:
This fixes an issue in `lg2 log` where `log` wouldn't parse any commandline arguments that were not `--`.

`match_arg_separator` matches only arguments that are `--` (so that `log --arg1 --arg2 -- thing1 thing2` stops parsing flags after `arg2`).